discovery-provider/src/tasks/tracks.py

```diff
@@ -83,6 +85,22 @@ def track_state_update(
 
                     track_events[track_id] = {"track": track_entry, "events": []}
 
+                track_metadata = None
+                if event_type in [
+                    track_event_types_lookup["new_track"],
+                    track_event_types_lookup["update_track"],
+                ]:
+                    track_metadata_digest = event_args._multihashDigest.hex()
+                    track_metadata_hash_fn = event_args._multihashHashFn
+                    buf = multihash.encode(
+                        bytes.fromhex(track_metadata_digest), track_metadata_hash_fn
+                    )
+                    cid = multihash.to_b58_string(buf)
+                    # do not process entry if cid is blacklisted
+                    if cid in blacklisted_cids:
```

Comment:
just so i understand, we no longer need this (https://github.com/AudiusProject/audius-protocol/pull/1778/files#diff-8b0220935e804df6c696aa7d91aa47fc8a06528112f172c75bb67664d366a185L406-L411) because here we skip to the next tx if the cid is in this set?

Reply:
yes, that is correct